relentless/optimize/method.py

```diff
@@ -37,16 +38,24 @@
 
 .. autoclass:: SteepestDescent
     :member-order: bysource
-    :members: optimize,
+    :members: descent_amount,
```

Comment:
Do you need to list all these members explicitly? (i.e., are there any you are excluding?) I think you can just use `:members:` to pull everything in by default.

Reply:
The properties weren't showing with just `:members:`.